lib/adapters/parsers/csv.js

```diff
@@ -9,6 +9,9 @@ class CSVParser extends base {
     constructor(options) {
         super(options);
         this.separator = options.separator ? options.separator : ',';
+        this.comment = options.comment ? options.comment : null;
```

Comment:
Then we'd have to rename `separator` to `separatorChar` ? I was keeping consistent with that specific option. @dmajda